lib/jekyll/commands/serve/servlet.rb

```diff
@@ -29,15 +29,15 @@ def chunked?
         end
 
         def inline?
-          @response["Content-Disposition"].start_with?("inline")
+          @response["Content-Disposition"].to_s.start_with?("inline")
```

Comment:
why would these ever not be strings?

Reply:
In these tests: https://travis-ci.org/jekyll/jekyll/builds/509863125

Reply:
Can we use `@response.fetch("Content-Disposition", "")` instead? `fetch` conveys the intent better by saying "we expect this to be here, but in case it's not, use this value instead"

Reply:
We can't use `#fetch` here because `@response` is an instance of [`WEBrick::HTTPResponse`](https://ruby-doc.org/stdlib-2.4.0/libdoc/webrick/rdoc/WEBrick/HTTPResponse.html) which doesn't respond to `#fetch`

Reply:
:sob: :sob: :sob: